_posts/2021-07-19-changelog-151.markdown

```diff
@@ -0,0 +1,148 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #151"
+date: 2021-07-19 12:00:00 +0300
+---
+
+
+## New Features
+
+* [#6392], [#7507], [#7461] Add `Unused import` inspection. The inspection helps you to avoid excessive and duplicating `use` declarations in your project.
+  Current restrictions:
+  * Only checks `use` declarations inside modules without a child module
+  * Currently, disabled by default due to possible false-positive warnings. You can enable it via `Preferences | Editor | Inspections` [settings](https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-code-analysis.html#inspection-settings) (by [@Kobzol])
+
+  {% include gif-img.html path="/assets/posts/changelog-151/unused-import" w="700px" %}
+
+* [#7432] Confirm loading untrusted Cargo project. Now when you try to open a Cargo project from an unknown source for the first time,
```

Comment:
I'd move this item at the beginning since it affects all users but `Unused import` inspection is disabled by default